src/components/PasswordInput.vue

```diff
@@ -66,18 +66,18 @@ export default defineComponent({
 
     blurInputPassword: function() :void {
       var passwdPattern = "^[^ ]{8,}$";
+      var sPass = this.modelValue.toString;
```

Comment:
what is this variable for ? it is unused.

Reply:
It was to let me use .length() but I found another easier way to do it. I can remove since its not necessary.